test/test_helper.rb

```diff
@@ -30,6 +30,38 @@
   ActiveRecord::Migration.execute "SET CONSTRAINTS ALL DEFERRED;"
 end
 
+module TestCaseRailsLoggerExtensions
+  def before_setup
+    super
+    @current_buffer = StringIO.new
+    @old_logger = Rails.logger
+    @old_ar_logger = ActiveRecord::Base.logger
+    Rails.logger = Foreman::SilencedLogger.new(ActiveSupport::TaggedLogging.new(Logger.new(@current_buffer)))
+    ActiveRecord::Base.logger = Rails.logger
+  end
+
+  def after_teardown
+    Rails.logger = @old_logger if @old_logger
+    ActiveRecord::Base.logger = @old_ar_logger if @old_ar_logger
+    if @current_buffer.nil?
+      Rails.logger.debug "WARNING: Looks like loggging teardown was removed by a test."
```

Comment:
This was a leftover when I had this hooked into setup/teardown framework. Removed.